src/Tribe/Tickets.php

```diff
@@ -295,13 +295,14 @@ public function get_event_reports_link( $post_id ) {}
 		public function get_ticket_reports_link( $post_id_deprecated, $ticket_id ) {}
 
 		/**
-		 * Returns a single ticket
+		 * Returns a single ticket.
 		 *
 		 * @abstract
 		 *
-		 * @param int $post_id ID of parent "event" post
-		 * @param int $ticket_id ID of ticket post
-		 * @return mixed
+		 * @param int $post_id   ID of parent "event" post.
+		 * @param int $ticket_id ID of ticket post.
+		 *
+		 * @return Tribe__Tickets__Ticket_Object|null
```

Comment:
:no_entry_sign: Function return type is not void, but function has no return statement

<sub>hash: ec3ee9383a5f9db2df16c0a321af36ba</sub>

Reply:
@sc0ttkclark True but IDE sees base/abstract method and not all those extending it so autocomplete doesn't work as it should without this. OK or not?

Reply:
We should return null or the ticket object if that's what we're doing. `return;` is void or a lack of return is void.

Reply:
Got it. Done.